.github/workflows/release.yaml

```diff
@@ -264,7 +264,11 @@ jobs:
       - uses: actions/checkout@v3
       - name: Create tutorials tar
         run: |
-          tar -cvf mlrun-tutorials.tar docs/tutorials
+          mkdir jupyter-welcome
```

Comment:
why create this dir instead of downloading the files into `docs/tutorials`?

Reply:
You are right that I can insert the new files into docs folder (not tutorials since we want README and welcome next to the tutorials folder and not inside).
@alonmr You think putting them in docs is better ?

Reply:
let's use `docs/jupyter-welcome` then. Just because I think that using the root directory is not safe since we don't know what it contains (since we're using github runners).
Also, use `mkdir -p`

Reply:
README and welcome moved inside docs folder